model/search_params_test.go

```diff
@@ -301,3 +304,99 @@ func TestParseSearchParams(t *testing.T) {
 		t.Fatalf("Incorrect output from parse search params: %v", sp)
 	}
 }
+
+func TestGetOnDateMillis(t *testing.T) {
+	for _, testCase := range []struct {
+		Input       string
+		StartOnDate int64
+		EndOnDate   int64
+	}{
+		{
+			Input:       "2018-08-01",
```

Comment:
I would like to have a "Name" explaining what the test is testing. something like "Well formated date" or "No date at all". This way we can know why is the test executing and why is passing or failing.